### PR TITLE
Update versions to latest - bump

### PR DESCRIPTION
**Pull request by SwayStar123**
Update forc to 0.60, fuels to 0.63, and fuel-core to 0.27

### CODE REVIEW

**Comment by bitzoic** [comment]
Will we need another update once a new version of forc is released for https://github.com/FuelLabs/fuel-bridge/pull/164?

**Review by DefiCake — APPROVED** [review]

**Review by K1-R1 — APPROVED** [review]

**Comment by DefiCake** [comment]
Please wait for @bitzoic  's approval before merging this.

**Review by bitzoic — APPROVED** [review]

**Pull request merged**
